cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -107,15 +107,24 @@ Whether they are FileSets or another type of dataset is handled by the CDAP runt
 .. highlight:: console
 
 However, you do need to tell CDAP the relative paths of the input and output files. Currently,
-this is only possible by specifying them as runtime arguments when the MapReduce program is started::
+this is only possible by specifying them as runtime arguments when the MapReduce program is started:
 
-  curl -v <base-url>/apps/FileSetExample/mapreduce/WordCount/start -d '{ \
-      "dataset.lines.input.paths":  "monday/my.txt", \
-      "dataset.counts.output.path": "monday/counts.out" }'
+.. tabbed-parsed-literal::
+
+  $ curl -w"\n" -X POST "http://example.com:10000/v3/namespaces/default/apps/FileSetExample/mapreduce/WordCount/start" \
+    -d "{ 'dataset.lines.input.paths':  'monday/my.txt', \
+          'dataset.counts.output.path': 'monday/counts.out' }"
```

Comment:
The outer quotes need to be double quotes. And then you need to escape all the inner double-quotes:
```
-d "'{ \"dataset.lines.input.paths\":  \"monday/my.txt\", \
           \"dataset.counts.output.path\": \"monday/counts.out\" }'"
```
To be honest, I haven't tested this for a bit,